lighthouse-cli/run.js

```diff
@@ -159,6 +162,12 @@ function saveResults(results, artifacts, flags) {
  * @return {!Promise<!LH.Results|void>}
  */
 function runLighthouse(url, flags, config) {
+  if (flags.reportMode) {
```

Comment:
this means you can't really do `-GAR` for the full thing and existence of `-R` forces report-only mode no matter what, is that what we want? I guess auditMode continues on to do what reportMode would do anyhow

Reply:
hmm that's true. let's talk about what's ideal here.